pkg/pillar/Dockerfile

```diff
@@ -1,11 +1,27 @@
 # Copyright (c) 2018 Zededa, Inc.
 # SPDX-License-Identifier: Apache-2.0
+
+FROM lfedge/eve-dom0-ztools:ff71338c79558620a3d6dd770ad0291b786e759d as zfs
+RUN mkdir /out
+# copy zfs-related files from dom0-ztools using prepared list of files
+RUN while read -r x; do \
+        if [ -d "$x" ]; then \
+          mkdir -p "/out/$x"; \
+        else \
+          cp -P "$x" "/out/$x"; \
+        fi \
+    done < /etc/zfs-files
+
 FROM lfedge/eve-alpine:3a7658b4168bcf40dfbcb15fbae8979d81efb6f1 as build
 
 ARG DEV=n
 
-ENV BUILD_PKGS git gcc linux-headers libc-dev make linux-pam-dev m4 findutils go util-linux make patch wget zfs-dev
-ENV PKGS alpine-baselayout musl-utils libtasn1-progs pciutils yajl xz bash iptables ip6tables iproute2 dhcpcd coreutils dmidecode libbz2 libuuid ipset curl radvd ethtool util-linux e2fsprogs libcrypto1.1 xorriso qemu-img jq e2fsprogs-extra keyutils ca-certificates ip6tables-openrc iptables-openrc ipset-openrc hdparm zfs
+ENV BUILD_PKGS git gcc linux-headers libc-dev make linux-pam-dev m4 findutils go util-linux make patch wget \
+    libintl libuuid libtirpc libblkid libcrypto1.1 zlib
+ENV PKGS alpine-baselayout musl-utils libtasn1-progs pciutils yajl xz bash iptables ip6tables iproute2 dhcpcd \
+    coreutils dmidecode libbz2 libuuid ipset curl radvd ethtool util-linux e2fsprogs libcrypto1.1 xorriso \
+    qemu-img jq e2fsprogs-extra keyutils ca-certificates ip6tables-openrc iptables-openrc ipset-openrc hdparm \
+    libintl libtirpc libblkid zlib
```

Comment:
Why do we need to add (libintl libtirpc libblkid zlib) as a result of using zfs built from source?

Reply:
I removed zfs from PKGS (as I build it from source) and now we should have dependencies of zfs package in place. For example you can find them [here](https://pkgs.alpinelinux.org/package/v3.13/main/x86_64/zfs) and [here](https://pkgs.alpinelinux.org/package/v3.13/main/x86_64/zfs-libs).